litex/soc/cores/cpu/rocket/core.py

```diff
@@ -141,7 +141,11 @@ def __init__(self, platform, variant="standard"):
             i_reset = ResetSignal("sys") | self.reset,
 
             # Debug (ignored).
-            i_resetctrl_hartIsInReset_0           = Open(),
+            #FIXME: dynamically add one of these for each core!!!
+            #i_resetctrl_hartIsInReset_0           = Open(),
+            #i_resetctrl_hartIsInReset_1           = Open(),
+            #i_resetctrl_hartIsInReset_2           = Open(),
+            #i_resetctrl_hartIsInReset_3           = Open(),
```

Comment:
@enjoy-digital: the rest of the rocket verilog interface stays the same, except for additional per-core instances of `i_resetctrl_hartIsInReset_X`.

Now, originally, this signal was commented out and left open *implicitly* until commit 116c2f154 where it was made explicit by being assigned with `Open()`.

What's a good way to handle a variable number of these signals (one per core)? Maybe something like using `self.cpu_params.update()` from a loop (`for i in range(num_cores)`)? Or simply commenting them back out and leaving them all open implicitly again?

Reply:
I fixed this in commit 3cb89bc by dynamically generating one signal per core. LMK what you think.